hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/security/token/block/BlockPoolTokenSecretManager.java

```diff
@@ -36,7 +38,7 @@
  */
 public class BlockPoolTokenSecretManager extends
     SecretManager<BlockTokenIdentifier> {
-  
```

Comment:
Can you go through all the files and remove the changes that do not pertain to the lines modified by this patch?